### ISSUE TITLE
Cannot start storybook on windows 10 - Error: Cannot find module 'node_modules\react-scripts\bin\react-scripts.js"\config\webpack.config'

### ISSUE THREAD

**Issue by reggieboyYEAH** [storybookjs/storybook#8838]
**Describe the bug**
Cannot start storybook on windows 10 when. When I run `npm run storybook`
it get this error

**To Reproduce**
1. Just npm run storybook

**Expected behavior**
It should just start storybook

**Screenshots**
![image](https://user-images.githubusercontent.com/7448059/68906764-83274100-0799-11ea-9cb2-e005ded660b5.png)

**Code snippets**
My package.json is here
https://github.com/reggieboyYEAH/modus-ui/blob/master/example/package.json

**System:**
Environment Info:

  System:
    OS: Windows 10
    CPU: (8) x64 Intel(R) Xeon(R) CPU E3-1241 v3 @ 3.50GHz
  Binaries:
    Node: 12.13.0 - C:\Program Files\nodejs\node.EXE
    npm: 6.12.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: 44.18362.329.0

**Additional context**
Weird thing is this exact configuration works on my mac.
I already tried to reinstall by
removing package.lock.json and deleting node_modules

**Comment by pocka** [comment]
Although not tested, I think [this line](https://github.com/storybookjs/storybook/blob/next/app/react/src/server/cra-config.ts#L38) produces wrong react-scripts' installation path (`node_modules\react-scripts\bin\react-scripts.js"`, trailing double-quotes is unnecessary) and it causes the error.
According to the comment in the file, that line will only run on Windows, so you don't see the error on macOS.

If my guess is correct, this patch solves the problem.
```diff
-/"\$basedir[\\/]([^\s]+?[\\/]bin[\\/]react-scripts\.js")/i
+/"\$basedir[\\/]([^\s]+?[\\/]bin[\\/]react-scripts\.js)"/i
```

NOTE: I tested `path.join` with a trailing double-quotes path on Linux, but it was resolved correctly 🤔  Is this behavior is Windows-specific one?
